frontend/src/app/pages/registration-done/registration-done.component.scss

```diff
@@ -0,0 +1,16 @@
+.nav-buttons {
+    margin-top: 2em;
+}
+
+.nav-button {
+    margin: 0 auto;
```

Comment:
Nemohlo by tady být `margin: 2em auto 0;`? Pak bychom nemuseli mít obalovací `<div class="nav-buttons">` s třídou navíc ;) 

Kdybys ale musel nechat parent třídu, tak já mám osobně radši když se to v SCSS zanořuje, jelikož je pak jasné, že se jedná o jeden logický blok, třeba: 
```
.nav-buttons {
    margin-top: 2em;

    button { ... }
}
```

Obecně se snažím držet co nejmíň tříd, ale když už musím použít víc vnořených do sebe, tak držím BEM konvenci pro názvy: https://cssguidelin.es/#bem-like-naming